iot/manager/manager.go

```diff
@@ -690,6 +652,166 @@ func sendCommand(w io.Writer, projectID string, region string, registry string,
 
 // [END iot_send_command]
 
+// START BETA FEATURES
+
+// [START iot_create_gateway]
+
+// createGateway creates a new IoT Core gateway with a given id, public key, and auth method.
+// gatewayAuthMethod can be one of: ASSOCIATION_ONLY, DEVICE_CREDENTIALS_ONLY, ASSOCIATION_AND_DEVICE_AUTH_TOKEN
```

Comment:
Period at the end. Is there a documentation link for the values of `gatewayAuthMethod`?